CODEOWNERS

```diff
@@ -16,13 +16,12 @@ Makefile                                 @terryzouhao
 /hypervisor/                             @anthonyzxu @dongyaozu
 /devicemodel/                            @anthonyzxu @ywan170
 /doc/                                    @dbkinder @deb-intel @NanlinXie
-/misc/tools/acrn-crashlog/               @chengangc @dizhang417 
-/misc/tools/acrnlog/                     @ywan170 @liuyuan-GitHub
-/misc/tools/acrntrace/                   @ywan170 @liuyuan-GitHub
-/misc/acrn-manager/                      @ywan170 @liuyuan-GitHub
-/misc/acrnbridge/                        @ywan170 @liuyuan-GitHub
-/misc/acrn-config/                       @terryzouhao
+/misc/debug_tools/acrn_crashlog/         @chengangc @dizhang417
+/misc/debug_tools/acrn_log/              @ywan170 @shuox
+/misc/debug_tools/acrn_trace/            @ywan170 @shuox
+/misc/services/acrn_manager/             @ywan170 @shuox
+/misc/services/acrn_bridge/              @ywan170 @shuox
```

Comment:
Do we have an owner we can already add for these folders?
* `misc/servcies/life_mngr`
* `misc/packaging`

Reply:
I will update that after I confirm the owner.